tpgtools/overrides/eventarc/beta/trigger.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- type: SET_HASH_FUNC
+  field: matching_criteria
+  details:
+    function: schema.HashResource(EventarcTriggerMatchingCriteriaSchema())
```

Comment:
Oh, I'm surprised that this isn't the default set func. I think it would be in MMv1, so it's probably a safe change to make?